### PR TITLE
Added Flutter SDK Documentation

### PR DESCRIPTION
**Pull request by desusai7**
## Added Flutter SDK Documentation

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: rudderstack-sdk-integration-guides/rudderstack-flutter-sdk.md [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+```
+
+* Add the following code somewhere in your application.
+
+```dart
+    RudderConfigBuilder builder = RudderConfigBuilder();
+    builder.withDataPlaneUrl(DATA_PLANE_URL);
+    builder.withTrackLifecycleEvents(true);
+    builder.withRecordScreenViews(true);
+    RudderClient.getInstance(WRITE_KEY,config: builder.build());
```
Comment (arnab-p) [inline rudderstack-sdk-integration-guides/rudderstack-flutter-sdk.md:66]: Format with 2-spaces and apply the same to other code snippets as well

File: rudderstack-sdk-integration-guides/rudderstack-flutter-sdk.md [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+      </td>
+      <td style="text-align:left"><code>string</code>
+      </td>
+      <td style="text-align:left">If you are using our open-source config generator, use this option to
+        point to your hosted <code>sourceConfig</code>. SDK will add <code>/sourceConfig</code> along
+        with this URL</td>
+      <td style="text-align:left"><code>https://api.rudderlabs.com</code>
+      </td>
+    </tr>
+  </tbody>
```
Comment (arnab-p) [inline rudderstack-sdk-integration-guides/rudderstack-flutter-sdk.md:392]: Can we convert this to markdown table?

**Review by arnab-p — CHANGES_REQUESTED** [review]
Add an entry to `SUMMARY.md` as well

File: rudderstack-sdk-integration-guides/rudderstack-flutter-sdk.md (hunk 2 above)
Comment (desusai7) [reply rudderstack-sdk-integration-guides/rudderstack-flutter-sdk.md:392]: Yeah sure

File: rudderstack-sdk-integration-guides/rudderstack-flutter-sdk.md (hunk 1 above)
Comment (desusai7) [reply rudderstack-sdk-integration-guides/rudderstack-flutter-sdk.md:66]: Will do as said

**Review by arnab-p — APPROVED** [review]

**Pull request merged**